tests/models/test_detection.py

```diff
@@ -0,0 +1,29 @@
+import torch
+import pytorch_lightning as pl
+
+from pl_bolts.models.detection import FasterRCNN
+from pl_bolts.datamodules import DummyDetectionDataset
+from torch.utils.data import DataLoader
+
+
+def _collate_fn(batch):
+    return tuple(zip(*batch))
+
+
+def test_fasterrcnn(tmpdir):
+
+    model = FasterRCNN()
+
+    image = torch.rand(1, 3, 400, 400)
+    model(image)
+
+
+def test_fasterrcnn_train(tmpdir):
+
+    model = FasterRCNN()
+
+    train_dl = DataLoader(DummyDetectionDataset(), collate_fn=_collate_fn)
```

Comment:
We have a dummy dataset already. why is this one different?

Reply:
Instead of a tensor label it returns a dict with values `{"boxes": FloatTensor[N, 4], "labels": Int64Tensor[N]}` as per the standard in pytorch for detection models. Should this be a choice in the existing dummy dataset?